src/test/java/Homework16.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.RemoteWebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+//    @Test
+//    static void Navigation() {
+//        WebDriver driver = new ChromeDriver();
+//        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+//
+//        String url = "https://bbb.testpro.io/";
+//
+//        driver.get(url);
+//
+//        WebElement registrationLink = driver.findElement(By.cssSelector("#hel"));
+//        registrationLink.click();
+//
+//        WebElement verifyRegistrationButton = driver.findElement(By.cssSelector("input[value='Register']"));
+//        Assert.assertTrue(verifyRegistrationButton.isDisplayed());
+//
+//        driver.quit();
+//    }
+
+}
```

Comment:
Uncomment the code or delete

![image](https://user-images.githubusercontent.com/104761482/219590281-49545ddb-a8bf-4f16-8ba3-c92661f033a0.png)